src/output.cpp

```diff
@@ -713,6 +713,16 @@ int query_int( const char *mes, ... )
     return atoi( raw_input.c_str() );
 }
 
+std::string query_string( const char *mes, ... )
```

Comment:
I thought to create a new function `query_string` performs quite similar to existing `query_int`.